.github/workflows/dependency-review.yml

```diff
@@ -13,6 +13,11 @@ on:
     branches:
       - main
 
+# Cancel any runs on the same branch
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Is the ref the commit? Or the branch?

Reply:
it should be branch according to https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs